src/fty_ambient_location_server.cc

```diff
@@ -461,7 +461,8 @@ s_ambloc_actor_stream (AmbientLocation* self, zmsg_t **message_p)
     }
     mtx_ambient_hashmap.lock();
     if (streq (fty_proto_operation (bmsg), FTY_PROTO_ASSET_OP_DELETE)
-                     || !streq (fty_proto_aux_string (bmsg, FTY_PROTO_ASSET_STATUS, "active"), "active") ) {
+                     || streq (fty_proto_aux_string (bmsg, FTY_PROTO_ASSET_STATUS, "active"), "inactive")
+                     || streq (fty_proto_aux_string (bmsg, FTY_PROTO_ASSET_STATUS, "active"), "retired")) {
```

Comment:
side note: in a soon future, we will want to also consider "maintenance" for the status